tests/ml_perf_end_to_end_tests.py

```diff
@@ -57,13 +57,21 @@ def test_all_mpl_approaches(self):
 
         exp = Experiment(is_save=False)
         # mpl_approaches = mpl.BASIC_MPL_APPROACHES.copy()
-        mpl_approaches = mpl.MULTI_PARTNER_LEARNING_APPROACHES
-
+        mpl_approaches = mpl.MULTI_PARTNER_LEARNING_APPROACHES.copy()
+        _ = mpl_approaches.pop("single-partner")
```

Comment:
I understand we can not test this method with the example below with 2 partners but we should nonetheless test it. I guess we would have to do a dedicated test with one partner.

Reply:
Good catch @RomainGoussault , I missed it but I agree with you. @arthurPignet is this something you could add before we merge?

Reply:
In fact it is tested within this function. See lines 66-73. I just cannot use the loop, because I want only one partner.